be/src/vec/sink/vrow_distribution.cpp

```diff
@@ -269,7 +269,6 @@ Status VRowDistribution::generate_rows_distribution(
     RETURN_IF_ERROR(_block_convertor->validate_and_convert_block(
             _state, &input_block, block, *_vec_output_expr_ctxs, input_rows, has_filtered_rows));
 
-    _tablet_finder->clear_for_new_batch();
```

Comment:
Why remove this?

Reply:
it will remove in vtablet_finder.cpp if needed